x-pack/legacy/plugins/console_extensions/spec/generated/security.delete_privileges.json

```diff
@@ -13,6 +13,6 @@
     "patterns": [
       "_security/privilege/{application}/{name}"
     ],
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-delete-privilege.html"
+    "documentation": "TODO"
```

Comment:
Let's manually revert this for now to not be blocked on upstream

Reply:
5b2c3eb